hypolith-hogsmeade.md

```diff
@@ -0,0 +1,11 @@
+# Hogsmeade
+
+### Restaurants
+- Three Broomsticks
+- Hogs Head
+
+### Attractions
+- Weasleys' Wizarding Weezes
+
```

Comment:
I'm pretty sure there is an Ollivander's outlet in Hogsmeade, however, might be wrong about that one.